warzone/TEMPFILE.cpp

```diff
@@ -0,0 +1,244 @@
+#include "PlayerStrategies.h";
```

Comment:
What is this file?

Reply:
I hate that they make us have all the strategies in a single file so when i work on it, i had all the HumanStrategy function definitions in here.